<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btn_ini">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止采集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭板卡</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="global_plt" native="true">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>9</y>
     <width>891</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="phase_plt" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>881</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="amplitude_plt" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>881</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_IQ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始解调</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_IQ_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭解调</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_alarm_pos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_alarm_pos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>位置确定</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
